src/main/bash/sdkman-init.sh

```diff
@@ -34,6 +34,11 @@ if [ -f "${SDKMAN_DIR}/etc/config" ]; then
 	source "${SDKMAN_DIR}/etc/config"
 fi
 
+# Load the sdkman user config if it exists.
+if [ -f "${HOME}/.config/sdkman/config" ]; then
```

Comment:
`"${XDG_CONFIG_HOME}/sdkman/config"`

if `XDG_CONFIG_HOME` is not set then `"${HOME}/.config/sdkman/config"`

https://wiki.archlinux.org/title/XDG_Base_Directory#User_directories